src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/SNSService.kt

```diff
@@ -141,6 +141,27 @@ class SNSActionPlanAppointmentService(
           mapOf("serviceUserCRN" to referral.serviceUserCRN, "referralId" to referral.id)
         )
 
+        snsPublisher.publish(referral.id, appointment.appointmentFeedbackSubmittedBy!!, snsEvent)
+      }
+      ActionPlanAppointmentEventType.SESSION_FEEDBACK_RECORDED -> {
+        val referral = event.deliverySession.referral
+        val appointment = event.deliverySession.currentAppointment
+          ?: throw RuntimeException("event triggered for session with no appointments")
+
+        val eventType = "intervention.session-appointment.session-feedback-submitted"
+
+        val snsEvent = EventDTO(
+          eventType,
+          "Session feedback submitted for a session appointment",
+          event.detailUrl,
+          appointment.appointmentFeedbackSubmittedAt!!,
+          mapOf(
+            "serviceUserCRN" to referral.serviceUserCRN,
+            "referralId" to referral.id,
+            "deliusAppointmentId" to appointment.deliusAppointmentId.toString()
```

Comment:
should we add this field to other appointment events?

Reply:
I see this as something internal to interventions but have put this on the message so that community-api has all the info it needs to update the correct appointment.